ocs_ci/ocs/resources/storage_cluster.py

```diff
@@ -384,18 +383,15 @@ def ocs_install_verification(
             kms = KMS.get_kms_deployment()
             kms.post_deploy_verification()
 
-    ocs_version = config.ENV_DATA["ocs_version"]
-    zone_num = utils.get_az_count()
-    if (
-        config.DEPLOYMENT.get("local_storage")
-        and Version.coerce(ocs_version) >= Version.coerce("4.7")
-        and zone_num < 3
-    ):
-        storage_cluster_obj = get_storage_cluster()
+    storage_cluster_obj = get_storage_cluster()
+    is_flexible_scaling = (
+        storage_cluster_obj.get()["items"][0].get("spec").get("flexibleScaling", False)
```

Comment:
Clusters with flexibleScaling disabled will not have this field in the yaml. Will is_flexible_scaling be set to false in such cases?

If this is meant to target the upgrade, we should also ensure that the flexible scaling field is not set on an upgraded cluster if it was not set in the earlier version.

Reply:
Clusters with flexibleScaling disabled will not have this field in the yaml. Will is_flexible_scaling be set to false in such cases? --> yes
`.get("flexibleScaling", False)` if flexibleScaling param doesn't exist is_flexible_scaling=False

If this is meant to target the upgrade, the test should also ensure that the flexible scaling field is not set on an upgraded cluster if it was not set in the earlier version. --> verify if flexibleScaling= True the failure domain is host (on upgrade and regular deployment)

Reply:
> If this is meant to target the upgrade, the test should also ensure that the flexible scaling field is not set on an upgraded cluster if it was not set in the earlier version. --> verify if flexibleScaling= True the failure domain is host (on upgrade and regular deployment)

What I mean is suppose the cluster did not have flexibleScaling = true before upgrade, it should not have flexibleScaling = true after upgrade. Is that being tested ?